addons/auth_oauth/models/res_users.py

```diff
@@ -23,9 +24,21 @@ class ResUsers(models.Model):
         ('uniq_users_oauth_provider_oauth_uid', 'unique(oauth_provider_id, oauth_uid)', 'OAuth UID must be unique per provider'),
     ]
 
-    @api.model
     def _auth_oauth_rpc(self, endpoint, access_token):
-        return requests.get(endpoint, params={'access_token': access_token}).json()
+        if self.env['ir.config.parameter'].sudo().get_param('auth_oauth_authorization_header'):
```

Comment:
```suggestion
        if self.env['ir.config_parameter'].sudo().get_param('auth_oauth_authorization_header'):
```

as pointed out by @Julien00859 :pray:

Reply:
fixed, also renamed the key to properly namespace it.